src/spdx3/model/spdx_document.py

```diff
@@ -27,9 +28,8 @@ class SpdxDocument(Bundle):
     # We overwrite the super-__init__ as check_types_and_set_values() takes care of all fields (including inherited ones).
     def __init__(self, spdx_id: str, creation_info: CreationInformation, name: str, elements: List[Element],
                  root_elements: List[Element], summary: Optional[str] = None, description: Optional[str] = None,
-                 comment: Optional[str] = None, verified_using: None = None, external_references: None = None,
-                 external_identifier: None = None, extension: None = None, namespace: Optional[NamespaceMap] = None,
-                 imports: Optional[List[ExternalMap]] = None, context: Optional[str] = None):
-
+                 comment: Optional[str] = None, verified_using: Optional[List[IntegrityMethod]] = None,
+                 external_references: None = None, external_identifier: None = None, extension: None = None,
+                 namespace: Optional[NamespaceMap] = None, imports: Optional[List[ExternalMap]] = None,
+                 context: Optional[str] = None):
```

Comment:
`verified_using` and `imports` should default to empty lists as always

Reply:
good catch